scalability_and_performance/cnf-tuning-low-latency-hosted-cp-nodes-with-perf-profile.adoc

```diff
@@ -0,0 +1,36 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="cnf-tuning-low-latency-hosted-cp-nodes-with-perf-profile"]
+= Tuning hosted control planes for low latency with the performance profile
+include::_attributes/common-attributes.adoc[]
+:context: cnf-low-latency-perf-profile-hosted-cp
+
+toc::[]
+
+Tune hosted control planes for low latency by applying a performance profile. With the performance profile you can restrict CPUs for infra and application containers, configure huge pages, Hyper-Threading, and configure CPU partitions for latency-sensitive processes.
+
+[id="cnf-create-performance-profiles-hosted-cp"]
+== Creating a performance profile for hosted control planes
+
+You can create a cluster performance profile by using the Performance Profile Creator (PPC) tool. The PPC is a function of the Node Tuning Operator. 
+
+The PPC combines information about your cluster with user-supplied configurations to generate a performance profile that is appropriate to your hardware, topology and use-case.
+
+The following is a high-level workflow for creating and applying a performance profile in your cluster:
+
```

Comment:
If this is a sequential workflow, then use an ordered, rather than unordered (bullets), list.